Manatee.Json.Tests.Benchmark/TestData/schema.json

```diff
@@ -0,0 +1,431 @@
+        "arrangements",
+        "reason"
+      ],
+      "properties": {
+        "poa_id": {
+          "type": "string"
+        },
+        "reason": {
+          "enum": [ "OWNER", "POA", "Poa", "GUARDIAN" ]
+        },
+        "permissions": {
+          "type": "array",
+          "items": { "$ref": "#/definitions/permission" }
+        },
+        "arrangements": {
+          "type": "array",
+          "items": { "$ref": "#/definitions/arrangement" }
+        }
+      }
+    },
+
+    "permission": {
+      "type": "object",
+      "required": [ "@type" ],
+      "properties": {
+        "@type": {
+          "enum": [
+            "KycReview",
+            "ViewPaymentDetails",
+            "SignPayment",
+            "CancelPayment",
+            "RegisterPayment",
+            "ViewAccountBalance",
+            "ViewAccountTransactions",
+            "ViewAccountInformation",
+            "ViewAccountCondition",
+            "RegisterAgreement",
+            "ViewAgreementDetails",
+            "SignAgreement",
+            "CancelAgreement"
+          ]
+        }
+      },
+      "allOf": [
+        {
+          "if": { "properties": { "@type": { "const": "KycReview" } } },
+          "then": {
+            "required": [ "profile" ],
+            "properties": {
+              "prod_base": {
+                "type": [ "string", "null" ],
+                "pattern": "^[0-9\\+]+$"
+              },
+              "profile": {
+                "type": "string",
+                "enum": [ "Administrator", "FullAuthorization", "SignAgreements" ]
+              }
+            }
+          }
+
+        },
+
+        {
+          "if": { "properties": { "@type": { "const": "ViewAccountBalance" } } },
+          "then": {
+            "properties": {
+              "prod_base": {
+                "type": [ "string", "null" ],
+                "pattern": "^[0-9\\+]+$"
+              }
+
+            }
+          }
+
+        },
+        {
+          "if": { "properties": { "@type": { "const": "ViewAccountTransactions" } } },
+          "then": {
+            "required": [ "reach" ],
+            "properties": {
+              "reach": { "enum": [ "International", "Domestic", "Internal" ] },
+              "prod_base": {
+                "type": [ "string", "null" ],
+                "pattern": "^[0-9\\+]+$"
+              }
+
+            }
+          }
+
+        },
+        {
+          "if": { "properties": { "@type": { "const": "ViewAccountInformation" } } },
+          "then": {
+            "properties": {
+              "prod_base": {
+                "type": [ "string", "null" ],
+                "pattern": "^[0-9\\+]+$"
+              }
+
+            }
+          }
+
+        },
+        {
+          "if": { "properties": { "@type": { "const": "ViewAccountCondition" } } },
+          "then": {
+            "prod_base": {
+              "type": [ "string", "null" ],
+              "pattern": "^[0-9\\+]+$"
+            }
```

Comment:
@idstam you have several places in your schema that call out what look like properties but outside of a property keyword.  Is this intentional?  These would be skipped by a validator, leaving only `{}` which will validate everything as passing.